plugin/rename.py

```diff
@@ -126,3 +141,45 @@ def on_prepare_result(self, response: Any, pos: int) -> None:
 
     def on_prepare_error(self, error: Any) -> None:
         sublime.error_message("Rename error: {}".format(error["message"]))
+
+    def _get_relative_path(self, file_path: str) -> str:
+        window = self.view.window()
+        if not window:
+            return file_path
+        base_dir = windows.lookup(window).get_project_path(file_path)
+        if base_dir:
+            return os.path.relpath(file_path, base_dir)
+        else:
+            return file_path
+
+    def _render_rename_panel(self, changes: Dict[str, List[TextEdit]], total_changes: int, file_count: int) -> None:
+        window = self.view.window()
+        if not window:
+            return
+        panel = ensure_rename_panel(window)
+        if not panel:
+            return
+        text = ''
+        for file, file_changes in changes.items():
+            text += '◌ {}:\n'.format(self._get_relative_path(file))
+            for edit in file_changes:
+                start = edit[0]
+                end = edit[1]
+                text += '\t{:>8}:{:<4} - {}:{}\n'.format(start[0] + 1, start[1] + 1, end[0] + 1, end[1] + 1)
```

Comment:
Showing only the starting point might be enough as it's looking a bit busy/confusing.